client/src/components/Workflow/Editor/Options.vue

```diff
@@ -55,7 +55,7 @@
                 ><span class="fa fa-floppy-o mr-1" />Save As...</b-dropdown-item
             >
             <b-dropdown-item href="#" @click="$emit('onLint')"
-                ><span class="fa fa-magic mr-1" />Check for Best Practices</b-dropdown-item
+                ><span class="fa fa-magic mr-1" />Validate</b-dropdown-item
```

Comment:
It isn't an invalid workflow because it doesn't have a license or an annotation - validate feels wrong? I think if you wanted to shorten this "Best Practices" might be a better shortening. I'm not going to die on that hill though.